CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        public IJokePrinter jokePrinter {get; private set;}
+        public IJokeService jokeService {get; private set;}
+
+        public Jester(IJokeService jokeService, IJokePrinter jokePrinter)
+        {
+            this.jokeService = jokeService ?? throw new ArgumentNullException();
+            this.jokePrinter = jokePrinter ?? throw new ArgumentNullException();
```

Comment:
When throwing ArgumentNullExceptions always specify the argument name
```suggestion
            this.jokeService = jokeService ?? throw new ArgumentNullException(nameof(jokeService));
            this.jokePrinter = jokePrinter ?? throw new ArgumentNullException(nameof(jokePrinter));
```